ansible/inventories/staging/group_vars/catalog-next/vars.yml

```diff
@@ -39,5 +39,5 @@ saml2_site_url: https://admin-catalog-next-datagov.dev-ocsit.bsp.gsa.gov/
 # Token for working with the Google Analytics API
 token_dat: "{{ catalog_next_ckan_token_dat }}"
 
-url_readonly: "{{ catalog_next_ckan_service_url }}"
-url_writable: "{{ catalog_next_ckan_admin_service_url }}"
+url_readonly: https://{{ catalog_host_public_next }}
```

Comment:
Don't these need to contain the `"` to encode the variable properly?

Reply:
Only if you start the value with {{}}. This value starts with `https://`, so it is fine.